CONTRIBUTING.md

```diff
@@ -0,0 +1,105 @@
+# Contributing
+
+**Table of Contents**
+1. [Welcome](#welcome) (2nd draft)
+2. [Philosophy](#philosophy) (TBD)
+3. [Reporting an Issue](#reporting-an-issue) (TBD)
+4. [Fixing an Issue (PRs)](#fixing-an-issue-prs) (2nd draft)
+5. [Setup and Run](#setup-and-run) (TBD)
+
+
+## Welcome
+Hi, and welcome to Highlight.js. We're a code-highlighting engine etc., but you probably know that. You want to contribute, and this document will be your guide.
+
+There are many ways to contribute to Highlight.js. Reporting issues is super useful. Feel free to work on an open issue. You can also help by writing [a plugin](https://highlightjs.readthedocs.io/en/latest/plugin-api.html), adding [a 3rd party grammar](https://github.com/highlightjs/highlight.js/blob/master/extra/3RD_PARTY_QUICK_START.md), or creating [a theme](https://highlightjs.readthedocs.io/en/latest/style-guide.html).
```

Comment:
>adding [a 3rd party grammar

There is also this page: https://highlightjs.readthedocs.io/en/latest/language-contribution.html
which is broader (it includes the link to the 3RD_PARTY_QUICK_START.md)

Reply:
Yes, there is already talk of merging them long-term in which case this obviously should link to the consolidated document.  This could mention both if it needed to until then - is that what you're suggesting?